core/src/org/labkey/core/view/template/bootstrap/ViewServiceImpl.java

```diff
@@ -488,8 +485,9 @@ public void renderPortalMenu(PrintWriter out, String title)
                 // Wrapped in immediately invoke function expression because of Issue 16953
                 if (PageFlowUtil.isPageAdminMode(getContext()))
                 {
-                    NavTree permissionsNav = new NavTree("Permissions",
-                            "javascript:LABKEY.Portal._showPermissions(" +
+                    NavTree permissionsNav = new NavTree("Permissions");
+                    permissionsNav.setScript(
+                            "LABKEY.Portal._showPermissions(" +
                                     config._webpart.getRowId() + "," +
                                     permissionString + "," +
                                     containerPathString + ");"
```

Comment:
I think this needs to return false. Clicking this adds to the browser history as it is.
```suggestion
                    permissionsNav.setScript(
                            "LABKEY.Portal._showPermissions(" +
                                    config._webpart.getRowId() + "," +
                                    permissionString + "," +
                                    containerPathString + "); return false;"
```